contributing/CODING_STANDARDS.md

```diff
@@ -118,6 +118,16 @@ inline namespace OPENASSETIO_CORE_ABI_VERSION {
 > macro forward declares using `class`, we must avoid using `struct`
 > to define classes or we risk linker errors on Windows.
 
+### Enumerations
+
+All enumerations in C++ should use scoped enumerations, i.e.
+`enum class`, to ensure constants do not pollute the parent namespace,
+and to add strong typing, avoiding a class of programmer error. This
+matches the [ISO C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class).
+
+Python bindings should therefore not use [export_values()](https://pybind11.readthedocs.io/en/stable/classes.html#enumerations-and-internal-types)
+when binding a scoped enum type, for consistency with C++.
```

Comment:
[fix] Could we add notes on what should be used instead?

Reply:
Updated in 1483f316 (as part of a rebase) - a bit tricky since there is nothing to use "instead", it's just about omitting a call.  Does the updated text clarify that?

Reply:
Spot on, thanks!